tests/chart_tests/test_astronomer_houston_sidecar.py

```diff
@@ -6,17 +6,17 @@
 chart_values = {
     "astronomer": {
         "houston": {
+            "command": ["/bin/sh"],
             "apiArgs": [
-                "sh",
                 "-c",
-                "yarn serve 1> >( tee -a /var/logs/houston/data.out.log ) 2> >( tee -a /var/logs/houston/data.err.log >&2 )",
+                "yarn serve 1> >( tee -a /var/log/houston/data.out.log ) 2> >( tee -a /var/log/houston/data.err.log >&2 )",
```

Comment:
good catch! [definitely should be `/var/log`](https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard#Directory_structure)